packages/ragbits-core/src/ragbits/core/cli.py

```diff
@@ -43,4 +61,34 @@ def generate_promptfoo_configs(
 
         generate_configs(file_pattern=file_pattern, root_path=root_path, target_path=target_path)
 
+    @prompts_app.command()
+    def render(prompt_path: str, payload: str | None = None) -> None:
+        """
+        Renders a prompt by loading a class from a module and initializing it with a given payload.
+        """
+        prompt = _render(prompt_path=prompt_path, payload=payload)
+
+        pprint("[orange3]RENDERED PROMPT:")
+        pprint(prompt.chat)
+
+    @prompts_app.command()
+    def execute(
```

Comment:
I called it `execute` instead of `exec`, because with `exec` I got `redefined-builtin` from `pylint` and I don't know if we want that

Reply:
You can use `@prompts_app.command(name="exec")` to override function name

Reply:
thanks, changed